src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -271,7 +285,7 @@ describe('form:', () => {
           validMigrations(formConfig);
           validObjectProperty(formConfig, 'chapters');
           validObjectProperty(formConfig, 'defaultDefinitions');
-          validFunctionProperty(formConfig, 'introduction', false);
+          validComponentProperty(formConfig, 'introduction', false);
```

Comment:
These are all properties than can be React components, and function checks don't cover all React components.